.github/workflows/ci.yml

```diff
@@ -12,12 +12,17 @@ on:
 
 jobs:
   build-test:
-    runs-on: windows-latest
-
     strategy:
       matrix:
-        version: [net462,net6.0,net7.0]
-
+        os: [ windows-latest, ubuntu-latest ]
+        version: [ net462, net6.0, net7.0 ]
+        exclude:
+        - os: ubuntu-latest
+          version: net462
+        - os: ubuntu-latest
+          version: net7.0
```

Comment:
I think you can remove this exclude.
Looks like the old Linux build was running net7

Reply:
Yes, thanks!